avocado/plugins/run.py

```diff
@@ -150,6 +153,15 @@ def configure(self, parser):
                              help="Inject [path:]key:node values into the "
                              "final multiplex tree.")
 
+    def _failfast(self, string):
+        if string == 'off':
+            return False
+        elif string == 'on':
+            return True
+        msg = ('Invalid --failfast option. Valid '
+               'options are: on, off (Defaults to on)')
```

Comment:
Nitpick: The default `choices` message is `avocado run: error: argument --failfast: invalid choice: foo (choose from 'on', 'off', None)`. Maybe we should use the same format (plus the additional info about default) (just an idea, no need to change it)